src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/UsecaseSelection.tsx

```diff
@@ -80,6 +80,7 @@ const UsecaseSelection = (props: { endTimeArray; setEndTimeArray; SREdata; setSR
     const response = await fetch(getListExperimentsURLWithParams(value));
     const data = await response.json();
     const experimentUsecase = data[0].experiment_usecase_type;
+    console.log(experimentUsecase);
```

Comment:
if not required can we remove the console statement.

Reply:
removed the console statement